src/shared/watchdog.c

```diff
@@ -15,9 +15,9 @@
 #include "watchdog.h"
 
 static int watchdog_fd = -1;
-static char *watchdog_device = NULL;
-static usec_t watchdog_timeout = USEC_INFINITY;
-static usec_t watchdog_last_ping = USEC_INFINITY;
+static char *watchdog_device;
+static usec_t watchdog_timeout;
+static usec_t watchdog_last_ping;
```

Comment:
i'd really initialize this one two. USEC_INFINITY sounds like the right value to initialize this to if we never pinged the device.

Reply:
With this change, `USEC_INIFINITY` means "enable the watchdog and leave the default programmed timeout whatever the value is". And I don't think that's the behavior we want by default as until now the watchdog is kept disabled by default.